packages/block-library/src/social-links/style.scss

```diff
@@ -17,18 +17,23 @@
 
 .wp-social-link {
 	display: block;
-	width: 36px;
-	height: 36px;
-	border-radius: 36px; // This makes it pill-shaped instead of oval, in cases where the image fed is not perfectly sized.
-	margin-right: 8px;
+	width: 2.25em;
+	height: 2.25em;
+	border-radius: 2.25em; // This makes it pill-shaped instead of oval, in cases where the image fed is not perfectly sized.
+	margin-right: 0.5em;
 	transition: transform 0.1s ease;
 	@include reduce-motion("transition");
 
 	a {
-		padding: 6px;
+		padding: 0.375em;
 		display: block;
 		line-height: 0;
 		transition: transform 0.1s ease;
+
+		svg {
+			width: 1.5em;
+			height: 1.5em;
+		}
```

Comment:
This rule isn't in `master`. Is this supposed to be here?

Reply:
Yeah, without these 2 lines the social icons are 24px. Which used to be fine with the previous styles since the rest of our styles were calculated around that `24px` value of the SVGs. But now that they are able to scale, the SVG needs to scale with them.
If we remove these 2 lines the result looks like this:

![Screenshot_2020-08-13 - lemon](https://user-images.githubusercontent.com/588688/90143647-89793680-dd86-11ea-875a-66a8369a7189.png)

whereas with these 2 lines included the result looks as it did before:

![Screenshot_2020-08-13 - lemon(1)](https://user-images.githubusercontent.com/588688/90143826-bd545c00-dd86-11ea-8486-9be94a1eccb4.png)